playground/TestShop/TestShop.AppHost/Program.cs

```diff
@@ -38,6 +38,10 @@
 
 builder.AddProject<Projects.MyFrontend>("frontend")
        .WithExternalHttpEndpoints()
+       .WithModifiedEndpoints(c =>
+       {
+           c.DisplayProperties.DisplayName = "TestShop frontend access";
+       })
```

Comment:
This seems like a bit of a sledge hammer to give names to endpoints from the launch profile.

@davidfowl Do you want to be able to give names to individual endpoints from the launch profile?

Reply:
I think so yes.